### ISSUE TITLE
[Feature Request]  Drow down menue for region content type

### ISSUE THREAD

**Issue by MartinW-S2M** [GraffitiProjectINDIGO/GRAPHIS#11]
Would be great to have also a drop down menue for region content type similar to region role

**Comment by MartinW-S2M** [comment]
This means also the config file structure needs to be changed. Where to get the possible entries for region role name and indentifier?

**Comment by BeyondConventionalBoundaries** [comment]
Indeed, now you populated the rectangles with textGraffito and the polygons with graffito. This behaviour is still OK (for us), but in the future, other people might like to use other stuff. So, the values can still be the auto-populated values like what GRAPHIS does now, but with a dropdown, we could then also change a rectangular region to graffito if we would once have a graffito whose outline is a perfect rectangle.

As before, we will incorporate many IPTC concepts in our GRAPHIS vocabulary, but for the moment we only have these two terms, that is correct.

- we have graffito (so ending on o); but the URI refers to graffiti (as this comes from our own thesaurus, where concepts are all plural: https://vocabs.acdh.oeaw.ac.at/graphis-imgreg/graffiti)
- text (graffito): https://vocabs.acdh.oeaw.ac.at/graphis-imgreg/textGraffito
